### PR TITLE
[dv] Support Xcelium and xprop in new dv regression flow

### PR DESCRIPTION
**Pull request by weicaiyang**
And also update vcs coverage and fix profile opt

Signed-off-by: Weicai Yang <weicai@google.com>

### CODE REVIEW

File: hw/dv/data/xcelium/xcelium.hjson [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+               "-64bit -xmlibdirname {build_dir}/xcelium.d -R",
+               "+SVSEED={seed}",
+               "+UVM_TESTNAME={uvm_test}",
+               "+UVM_TEST_SEQ={uvm_test_seq}"]
+
+  build_modes: [
+    {
+      name: xcelium_waves
+      is_sim_mode: 1
+      build_opts: []
```
Comment (sriyerg) [inline hw/dv/data/xcelium/xcelium.hjson]: You can remove these empty lists.

File: hw/dv/data/vcs/vcs.hjson [hunk 2]
```diff
@@ -34,7 +34,7 @@
 
   // Defaults for VCS
   cov_metrics:          "line+cond+fsm+tgl+branch+assert"
-  vcs_cov_hier:         ""
+  vcs_cov_hier:         "-cm_hier {proj_root}/hw/dv/tools/vcs/cover.cfg"
```
Comment (sriyerg) [inline hw/dv/data/vcs/vcs.hjson]: Need to find a way to override this variable if user desires to change this 'default' to something else.

File: hw/dv/data/xcelium/xcelium.hjson (hunk 1 above)
Comment (weicaiyang) [reply hw/dv/data/xcelium/xcelium.hjson]: Updated

File: hw/dv/data/vcs/vcs.hjson (hunk 2 above)
Comment (weicaiyang) [reply hw/dv/data/vcs/vcs.hjson]: Agree. Will figure it out later.

**Pull request merged**